layers/osx/README.org

```diff
@@ -32,11 +32,7 @@ and we encourage you to do so :)
 
 ** Layer
 
-To use this configuration layer, add it to your =~/.spacemacs=
-
-#+BEGIN_SRC emacs-lisp
-  (setq-default dotspacemacs-configuration-layers '(osx))
-#+END_SRC
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to add `osx` to the existing list of `dotspacemacs-configuration-layers` in this file.
```

Comment:
Cool! That makes sense, thanks for giving me a better idea of how the
workflow for this project works! <3

On Tue, Jan 5, 2016 at 4:00 PM, Fabien Dubosson <notifications@github.com>
wrote:

> In layers/osx/README.org
> <https://github.com/syl20bnr/spacemacs/pull/4431#discussion_r48902246>:
>
> > @@ -32,11 +32,7 @@ and we encourage you to do so :)
> >
> >  ** Layer
> >
> > -To use this configuration layer, add it to your =~/.spacemacs=
> > -
> > -#+BEGIN_SRC emacs-lisp
> > -  (setq-default dotspacemacs-configuration-layers '(osx))
> > -#+END_SRC
> > +To use this configuration layer, add it to your =~/.spacemacs=. You will need to add `osx` to the existing list of `dotspacemacs-configuration-layers` in this file.
>
> The commits that are done now will be available in the 0.106 release
> (except if @syl20bnr <https://github.com/syl20bnr> want to hotfix them),
> what means your change will only be visible by people using the develop
> branch, and those ones know how to use layers [image: :smiley:] There
> should be enough time before the 0.106 release to solve #2213
> <https://github.com/syl20bnr/spacemacs/pull/2213>. As you said it's
> problematic for the user, so I milestoned it so we don't forget to look at
> it :-) Keep this issue open… just in case.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/4431/files#r48902246>.
>